steps/engine/clusterloader2/slo/collect.yml

```
@@ -16,8 +16,8 @@ steps:
     set -eo pipefail
 
     PYTHONPATH=$PYTHONPATH:$(pwd) python3 $PYTHON_SCRIPT_FILE collect \
-      $CPU_PER_NODE $NODE_COUNT $MAX_PODS $REPEATS \
-      $CL2_REPORT_DIR "$CLOUD_INFO" $RUN_ID $RUN_URL $SERVICE_TEST $TEST_RESULTS_FILE \
+      $CPU_PER_NODE $NODE_COUNT ${MAX_PODS:-0} \
```

I think just `$MAX_PODS` is good enough as I believe other pipelines already use the value

---

Oh so I kept it as that in case in the cnp/ccnp pipeline the max pods value is left blank in the variables in the pipeline. In cnp-ccnp-feature.yaml, I take in the variable from the pipeline into max_pods and I had done testing if any of the values are blank, I found that it causes an error when hitting the execute or collect file so I just did it for preventative measure for precaution. Would it be okay to keep it?

---

Yeah I guess it's fine to keep it now. But in the future, we should enforce this value to used by all tests so we don't have different hard-coded value anymore, and failure to set it should result in pipeline failure